elasticsearch_dsl/search.py

```diff
@@ -382,6 +386,38 @@ def script_fields(self, **kwargs):
         s._script_fields.update(kwargs)
         return s
 
+    def source(self, source=None):
+        """
+        Selectively control how the _source field is returned.
+
+        :arg source: wildcard string, array of wildcards, or dictionary of includes and excludes
+
+        If ``source`` is None, the entire document will be returned for
+        each hit.  If source is a wildcard or array of wildcards, matching
+        fields will be included. If source is a dictionary with keys of 'include' and/or
+        'exclude' the fields will be either included or excluded appropriately.
+
+        Calling this multiple times will override the previous values with the new ones.
+
+        Example::
+
+            s = Search()
+            s = s.source("obj.*")
+
+            s = Search()
+            s = s.source(["obj1.*", "obj2.*"])
+
+            s = Search()
+            s = s.source({
+                "include": ["obj1.*", "obj2.*"],
+                "exclude": ["*.description"]
```

Comment:
this seems cumbersome, how would you feel about using kwargs instead?
```
s = Search()
s = s.source(include=['obj1.*'], exclude=["*.description"])
```
with only the keyword specified being overridden, meaning the example could also be written as:
```
s = Search()
s = s.source(include=['obj1.*']).source(exclude=["*.description"])
```